awx_collection/README.md

```diff
@@ -27,16 +46,6 @@ The following notes are changes that may require changes to playbooks:
  - The `variables` parameter in the `tower_group`, `tower_host` and `tower_inventory` modules are now in `dict` format and no longer supports the use of the `C(@)` syntax (for an external `vars` file).
  - Some return values (e.g., `credential_type`) have been removed. Use of `id` is recommended.
 
-## Running
-
-To use this collection, the "old" `tower-cli` needs to be installed
-in the virtual environment where the collection runs.
-You can install it from [PyPI](https://pypi.org/project/ansible-tower-cli/).
-
-To use this collection in AWX, you should create a custom virtual environment into which to install the requirements. NOTE: running locally, you will also need
-to set the job template `extra_vars` to include `ansible_python_interpreter`
-to be the Python in that virtual environment.
```

Comment:
@beeankha I realized that these instructions are incomplete. For instance, how do you install the collection on the server? How do you set credentials? You can write a playbook to run a playbook inside of Tower/AWX. But that's too much for these docs.

Reply:
Hoping to have more resources soon to solve this problem... 😄